source/includes/_prices.md

```diff
@@ -20,22 +22,128 @@ curl "http://api.lettuce.money/prices?to_symbol=ZAR&from_symbols=BTC,USD,MSFT"
   "fromSymbols": [
     {
       "fromSymbol": "BTC",
-      "lastUpdated": "2018-12-31T11:59:01.001Z",
-      "price": 0.000021,
-      "source": "Crypto Compare"
+      "price": 53504.54788657036,
+      "source": "cryptocompare"
+    },
+    {
+      "fromSymbol":"ETH",
+      "price":1841.2815319462345,
+      "source":"cryptocompare"
+    },
+    {
+      "fromSymbol":"XMR",
+      "price":667.5567423230975,
+      "source":"cryptocompare"
+    }
+  ],
+  "errors": []
+}
+```
+
+## Stocks
+
```

Comment:
Again, I'm not convinced that we should split by asset type.

Reply:
We discussed and agreed that this might be useful in future for debugging. Leaving it ini for now.